### DISCUSSION TITLE
How to disable update notification

### DISCUSSION THREAD

**Discussion by gsal** [VSCodium/vscodium#1189] [General]
I have installed codium in Linux, in a central location for everyone to use; needless to say, only root has write permission to such spot. 

Since users cannot really update codium, I would very much like to turn off the "There is an update available " notification; a google search yields user preference settings, but I am looking for a way to set that at the installation level.

is there a way to do this?

**Comment by joecool1029** [comment]
Looking into this as well for Gentoo since portage manages our updates. It looks like resources/app/product.json file would be where to modify. Just remove the “updateUrl” line with sed or something after installing and updater should be disabled.

**Reply by joecool1029** [reply]
``` 
sed -i "/updateUrl/d" resources/app/product.json 
```

Add this somewhere in your deployment script. I've submitted a PR for gentoo.

**Comment by daiyam** [comment]
use `"update.mode": "none",` in your `settings.json`....

**Reply by joecool1029** [reply]
Not the best idea for centralized deployment/management of updates. If the distro or a company's IT management wants the responsibility of handling updates, it's better to disable the server at package level and not risk breaking user config.

**Reply by daiyam** [reply]
Yeah, I would apply both.

**Reply by joecool1029** [reply]
I would recommend users apply the second as well, but ultimately it's not a distro's business to mess around with local user config files on every upgrade. Is there a second location we could apply that setting globally in the application directory?
